src/rpcq.lisp

```diff
@@ -403,8 +401,7 @@ CoreMessages.~A = _deprecated_property(~:*~A)
 
 (defun deserialize (payload)
   "Deserialize the object(s) encoded in PAYLOAD (string or stream)."
-  (%deserialize (let ((*read-default-float-format* 'double-float))
-                  (yason:parse payload :json-arrays-as-vectors t))))
+  (%deserialize (messagepack:decode payload)))
```

Comment:
If we want to be true to the docstring, we would need to do a typecase, as `stream` decoding is done with `decode-stream` AFAICT